features/home/feed.tsx

```diff
@@ -202,6 +202,16 @@ export const Feed = () => {
                     }}
                   />
                 ))}
+                
+                {/* Logout button at the bottom of the feed */}
```

Comment:
Why do we need this? The logout button is on the profile - if we have an infinite scrolling feed then the users won't see this anyway